wgpu-hal/src/dx12/command.rs

```diff
@@ -754,12 +756,23 @@ impl crate::CommandEncoder for super::CommandEncoder {
             if let Some(ds_view) = ds_view {
                 if flags != Direct3D12::D3D12_CLEAR_FLAGS::default() {
                     unsafe {
-                        list.ClearDepthStencilView(
-                            ds_view,
+                        // list.ClearDepthStencilView(
+                        //     ds_view,
+                        //     flags,
+                        //     ds.clear_value.0,
+                        //     ds.clear_value.1 as u8,
+                        //     None,
+                        // )
+                        // TODO: Replace with the above in the next breaking windows-rs release,
+                        // when https://github.com/microsoft/win32metadata/pull/1971 is in.
+                        (windows_core::Interface::vtable(list).ClearDepthStencilView)(
+                            windows_core::Interface::as_raw(list),
+                            mem::transmute(ds_view),
                             flags,
                             ds.clear_value.0,
                             ds.clear_value.1 as u8,
-                            &[],
+                            0,
+                            std::ptr::null(),
                         )
```

Comment:
With this, confirmed with others' reports, we can now safely call "Fixes https://github.com/gfx-rs/wgpu/issues/6162".